discussions/api.py

```diff
@@ -97,6 +97,8 @@ def create_discussion_user(discussion_user):
     api = get_staff_client()
     result = api.users.create(
         name=profile.full_name,
+        email=profile.user.email,
+        email_optin=profile.email_optin,
         image=profile.image.url if profile.image else None,
```

Comment:
Sorry I missed this on the ticket, but the profile fields have been refactored in `open-discussion-client` to a single `profile=` arg. See https://github.com/mitodl/open-discussions-client/blob/master/open_discussions_api/users/client_test.py#L26-L35

Please also functionally test this code against open-discussions before putting it back in review.

Reply:
@rhysyngsun can you give me overview how to test it?